lib/src/http_unmodifiable_map.dart

```diff
@@ -53,6 +53,7 @@ class HttpUnmodifiableMap<V> extends UnmodifiableMapView<String, V> {
 /// A const implementation of an empty [HttpUnmodifiableMap].
 class _EmptyHttpUnmodifiableMap<V> extends MapView<String, V>
     implements HttpUnmodifiableMap<V> {
-  bool get _ignoreKeyCase => true;
   const _EmptyHttpUnmodifiableMap() : super(const {});
+
+  bool get _ignoreKeyCase => true;
```

Comment:
Yes. In general, please don't change things here just because lints say to—this package doesn't use the linter and it causes churn.